firebase-firestore/src/roboUtil/java/com/google/firebase/firestore/ValueHelper.java

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Google LLC
```

Comment:
Note that you currently only need this for tests, but eventually, won't the UserDataConverter need this as well? Maybe put this in with regular sources?

Reply:
I'd like to punt on this for now to figure out how much of this code I will need in UserDataConverter and where to put it. I added a TODO.

Reply:
FWIW, I think it's nice that this is a separate class that just has these static, simple functions. Moving it into the `UserDataConverter` would muddy that considerably. I was proposing just moving the class wholesale into the same package, or maybe even merging with/renaming `ProtoValues` and putting it in `model`.

In any case, no action required for this PR for sure.